### ISSUE TITLE
Cannot manually update INI file to change settings 

### ISSUE THREAD

**Issue by XiaoheLiu** [texstudio-org/texstudio#1235]
### Environment
- TeXstudio: 3.0.0
- Qt: 5.15.0, compiled with Qt 5.15.0 R
- OS: Mac
- TeX distribution: texlive

### Expected behavior

1. Close TexStudio.
2. Open the `texstudio.ini` file and replace the `[format]` section with a different color scheme. I used [this Solarized scheme](https://gitlab.com/adin/texstudio-profiles/-/blob/master/solarized.txsprofile) for example.
3. Reopen TexStudio, and the color scheme should be changed to the chosen scheme.

### Actual behavior
In the 3rd step, instead of seeing the updated color scheme, the color scheme is reset to the default from the software. The `texstudio.ini` file is overwritten by the default configurations again. 

### How to reproduce
Follow the steps in the "expected behavior" section.

P.s. this issue does not present in older versions (2.12.X).

**Comment by muzimuzhi** [comment]
With 3.0.0 on macOS 10.15.6, I cannot reproduce your problem.

In the 3rd step, the updated color scheme is applied, and the `.ini` file is not overwritten, apart from re-sorting and insertion of `xxx\xxx\wrapAround=false` lines.

Did you (i) successfully exit/quit the texstudio, and (ii) save the `.ini` file?

**Comment by XiaoheLiu** [comment]
Hi, thanks for looking into this.
I am using 3.0.0 and macOS 10.15.6, too, but the problem still occurs. 
Yes (i) I did quit texstudio properly by hitting the "quit texstudio" from the menu. I even restarted my computer and made sure texstudio is not opened at all. and (ii) I did save the `~/.config/texstudio/texstudio.ini` file. Actually, I copied and pasted my `.ini` file into the `~/.config/texstudio/` directory and have opened the modified file to check it was correct. But after I started texstudio, the `.ini` file is overwritten by the default.

Below is the setting file that I tried to use. I only changed the `[format]` section. (The extension is changed to `.txt` due to Github uploading restrictions)
[texstudio.txt](https://github.com/texstudio-org/texstudio/files/5144911/texstudio.txt)

**Comment by muzimuzhi** [comment]
By replacing local `texstudio.ini` by the provided `texstudio.txt`, everything looks good.
![image](https://user-images.githubusercontent.com/6376638/91640162-117d6280-ea4e-11ea-9404-3511f8f5b5e2.png)

@sunderme Does texstudio write any log when processing `texstudio.ini`?

**Comment by muzimuzhi** [comment]
Guess the related source lines are
https://github.com/texstudio-org/texstudio/blob/4d95d467b01920180640c2d6e73ab1c3cd90a350/src/configmanager.cpp#L800-L815

**Comment by sunderme** [comment]
> By replacing local `texstudio.ini` by the provided `texstudio.txt`, everything looks good.
> ![image](https://user-images.githubusercontent.com/6376638/91640162-117d6280-ea4e-11ea-9404-3511f8f5b5e2.png)
> 
> @sunderme Does texstudio write any log when processing `texstudio.ini`?

No log

**Comment by XiaoheLiu** [comment]
Hi @muzimuzhi , I think I got the problem solved!

I deleted my `~/.config/texstudio/` folder and texstudio, and then installed it again. This time I found that there is only a `[formatsDark]` section but no `[format]` section in the `texstudio.ini`.  This might be because my mac is in the dark theme, and texstudio automatically adjusted itself to be in dark theme. I added my color theme settings to the `[formatsDark]` section and the color theme was adjusted properly.
